docs/source/conf.py

```diff
@@ -0,0 +1,179 @@
+# Copyright (c) Microsoft Corporation
+# Licensed under the MIT License.
+#
+# Configuration file for the Sphinx documentation builder.
+#
+# This file only contains a selection of the most common options. For a full
+# list see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+# -- Path setup --------------------------------------------------------------
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+import inspect
+import os
+import raiwidgets
+import sys
+sys.path.insert(0, os.path.abspath(os.path.join('..', 'raiwidgets')))
+
+
+# -- Project information -----------------------------------------------------
+
+project = 'Responsible AI Widgets'
+copyright = 'Microsoft Corporation'
+author = 'Roman Lutz, Ilya Matiach, Ke Xu'
+
+# The full version, including alpha/beta/rc tags
+release = 'v0.2.0'
+
+
+# -- General configuration ---------------------------------------------------
+
+# Add any Sphinx extension module names here, as strings. They can be
+# extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
+# ones.
+extensions = [
+    'sphinx.ext.autodoc',
+    'sphinx.ext.doctest',
+    'sphinx.ext.extlinks',
+    'sphinx.ext.intersphinx',
+    'sphinx.ext.linkcode',
+    'sphinx.ext.mathjax',
+    'sphinx.ext.napoleon'
+]
+
+# Add any paths that contain templates here, relative to this directory.
+templates_path = ['_templates']
+
+# List of patterns, relative to source directory, that match files and
+# directories to ignore when looking for source files.
+# This pattern also affects html_static_path and html_extra_path.
+exclude_patterns = []
+
+
+intersphinx_mapping = {
+    'python3': ('https://docs.python.org/3', None),
+    'numpy': ('https://numpy.org/doc/stable/', None),
+    'pandas': ('https://pandas.pydata.org/pandas-docs/stable/', None),
+    'sklearn': ('https://scikit-learn.org/stable/', None),
+    'fairlearn': ('https://fairlearn.github.io/main/', None),
+}
+
+
+# -- Options for HTML output -------------------------------------------------
+
+# The theme to use for HTML and HTML Help pages.  See the documentation for
+# a list of builtin themes.
+#
+html_theme = 'pydata_sphinx_theme'
+
+# Theme options are theme-specific and customize the look and feel of a theme
+# further.  For a list of options available for each theme, see the
+# documentation.
+html_theme_options = {
+    # TODO: fork the pydata-sphinx-theme to integrate these with logo
+    "external_links": [
+        # {"name": "StackOverflow",
+        #  "url": "https://stackoverflow.com/questions/tagged/fairlearn"}
```

Comment:
remove fairlearn commented out stuff here?

Reply:
I want to eventually add StackOverflow, but currently there's no tag for raiwidgets and I don't have permissions to add such a tag on StackOverflow :-) I'll change the tag to raiwidgets in the text, though.